olp/utils.py

```diff
@@ -83,6 +83,29 @@ def remove_perm(user, permission, obj=None):
     return True
 
 
+def remove_permission_for_object(model_instance):
+    """
+    Removes a permission for a given object.
+    """
+
+    from django.contrib.contenttypes.models import ContentType
+    from olp.models import ObjectPermission
+
+    # Get the content type for the given instance
+    object_content_type = ContentType.objects.get_for_model(
+        model_instance.__class__
```

Comment:
You should be able to safely pass in a model instance, `get_for_model` works for [both classes and instances](https://docs.djangoproject.com/en/1.8/ref/contrib/contenttypes/#django.contrib.contenttypes.models.ContentTypeManager.get_for_model).